sass/discord.css

```diff
@@ -0,0 +1,56 @@
+#conversejs.theme-discord {
```

Comment:
I think we should not use the name `discord` anywhere. You can come up with a new name, or we simply call it the "dark" theme.